src/components/Forms/NewEvent.js

```diff
@@ -48,6 +52,50 @@ export default function NewEventForm(props) {
         return (errors[id] && submitCount > 0) || (touched[id] ? errors[id] : "")
     }
 
+    const handleInfoClick = () => {
+        window.alert(`Longitude and latitude are used for routing purposes for the mobile app. These values can be found on Google Maps by right clicking any location and pressing "What's here?"
```

Comment:
Don't think we've been including `window` when we use `alert`

```suggestion
        alert(`Longitude and latitude are used for routing purposes for the mobile app. These values can be found on Google Maps by right clicking any location and pressing "What's here?"
```